src/Controller/AiSearchBlockController.php

```
@@ -135,4 +137,89 @@ public function search(Request $request) {
+    if (!$view) {
+      return new JsonResponse(['html' => '<p>Error: Could not load view.</p>']);
+    }
+
+    // Use the correct display.
+    $view->setDisplay($display_id);
+
+    // ----- Exposed input (optional) -----
+    // Try to find the exposed fulltext identifier; default to 'search_api_fulltext'.
+    $filters = $view->display_handler->getOption('filters') ?: [];
+    $filter_key = 'search_api_fulltext';
+    foreach ($filters as $id => $filter) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$id'. (Unused Code Rules)

(UnusedLocalVariable)